README.md

```
@@ -17,4 +17,119 @@
 ![Supabase](https://img.shields.io/badge/Supabase-PostgreSQL-brightgreen?logo=supabase)
 ![REST API](https://img.shields.io/badge/REST-API-red?logo=api&logoColor=white)
 ![Android Studio](https://img.shields.io/badge/Android_Studio-Giraffe-green?logo=androidstudio&logoColor=white)
-![License](https://img.shields.io/badge/License-MIT-yellow.svg)
\ No newline at end of file
+![License](https://img.shields.io/badge/License-MIT-yellow.svg)
+
+## **📖 About EduBridge**  
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)